### PR TITLE
Part3

### PR DESCRIPTION
**Pull request by GlizzyProductions**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -142,27 +149,35 @@ struct MassageChair
...
 }
 
 double MassageChair::displayTimer(double msgDuration)
 {
     double chairClock = 0;
     char durationSelected = 'a';
-    double countdown = 30.00;
-    msgDuration = countdown;
+    double countdown = 0;
+    countdown = msgDuration;
```
Comment (matkatmusic) [inline main.cpp]: What is the point of initializing a variable to `0` and then assigning a value to it immediately after?
`auto countdown = msgDuration;`
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -275,16 +323,31 @@ struct Human
...
+}
+
+Human::Human()
+{
+    std::cout << "Human being constructed!" << std::endl;
+}
+
 void Human::HealthStatus::contractSTD(std::string whichSTD, std::string dateContracted)
 {
     whichSTD = "None";
```
Comment (matkatmusic) [inline main.cpp]: Don't overwrite function arguments.
Function arguments convey information. When you overwrite them like you were doing here, erase that information. Do something else in this function to clear the warning about the unused variable.
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -364,11 +435,17 @@ struct PhysicalAttribrutes
...
+    void loseLimb();
 };
 
+PhysicalAttribrutes::PhysicalAttribrutes()
+{
+    std::cout << "PhysicalAttribrutes is being constructed!" << std::endl;
+}
 void PhysicalAttribrutes::selectHairColor(std::string colorChoice)
 {
     colorChoice = "New Color";
```
Comment (matkatmusic) [inline main.cpp]: Don't overwrite function arguments.
Function arguments convey information. When you overwrite them like you were doing here, erase that information. Do something else in this function to clear the warning about the unused variable.
- [x] revise

File: main.cpp [hunk 4]
```diff
@@ -392,12 +470,19 @@ struct Education
...
 };
 
+Education::Education()
+{
+    std::cout << "Education being constructed!" << std::endl;
+}
+
 std::string Education::addDegree(std::string newestDegree)
 {
     newestDegree = "Degree Name";
```
Comment (matkatmusic) [inline main.cpp]: Don't overwrite function arguments.
Function arguments convey information. When you overwrite them like you were doing here, erase that information. Do something else in this function to clear the warning about the unused variable.
- [x] revise

File: main.cpp [hunk 5]
```diff
@@ -424,26 +508,33 @@ struct Home
...
 {
-    
+    ++numOfRooms;
+    std::cout << "Congratulations!! you have just added a room to your house! you now have " << numOfRooms << "rooms\n";
 }
 
 std::string Home::deteriorateOverTime(std::string brokenItem)
 {
     brokenItem = "Item that is broken";
     return brokenItem;
```
Comment (matkatmusic) [inline main.cpp]: Don't overwrite function arguments.
Function arguments convey information. When you overwrite them like you were doing here, erase that information. Do something else in this function to clear the warning about the unused variable.
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 2 above)
Comment (GlizzyProductions) [reply main.cpp]: I understand, I actually started wondering about this as I moved through this part of the project.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**